lib/if.h

```diff
@@ -411,6 +411,7 @@ struct connected {
 #define ZEBRA_IFA_SECONDARY    (1 << 0)
 #define ZEBRA_IFA_PEER         (1 << 1)
 #define ZEBRA_IFA_UNNUMBERED   (1 << 2)
+#define ZEBRA_IFA_DADFAILED    (1 << 3)
```

Comment:
This might be functionally identical to / duplicate with `!ZEBRA_IFC_REAL`/`ZEBRA_IFC_QUEUED`.  I think it might be better if we don't create a new flag for this and instead put a "unusable reason" in there separately.

Reply:
Btw: despite the name, `ZEBRA_IFC_QUEUED` is very much intended for this use; I remember that @cfra actually worked on this a few years back.  It's named `QUEUED` since the primary use was the DAD _delay_ after adding an address until it becomes usable, but that's functionally pretty much equivalent to DAD failure.

Reply:
QUEUED means queued for data plane operations.  This is not the right thing to call it.

Reply:
QUEUED should match the TENTATIVE flag right?  This is different right?